packages/expo-audio/src/AudioModule.types.ts

```diff
@@ -90,6 +90,11 @@ export declare class AudioPlayer extends SharedObject<AudioEvents> {
    */
   shouldCorrectPitch: boolean;
 
+  /**
+   * The current status of the audio player.
```

Comment:
I'm wondering whether to document this property – in theory everything can be accessed by individual properties, so it's basically duplicating.

Reply:
I was thinking the same, I'll add `@hidden`